packages/wallet/src/redux/store.ts

```diff
@@ -1,16 +1,33 @@
 import { applyMiddleware, compose, createStore } from 'redux';
 import { fork } from 'redux-saga/effects';
 import createSagaMiddleware from 'redux-saga';
-
+import * as storage from 'redux-storage';
 const sagaMiddleware = createSagaMiddleware();
 
 import { walletReducer } from './reducer';
 import { sagaManager } from './sagas/saga-manager';
+import filter from 'redux-storage-decorator-filter';
+import createEngine from 'redux-storage-engine-indexed-db';
+
+// We currently whitelist the values that we store/load.
+const storageEngine = filter(createEngine('magmo-wallet'), [
+  'whitelisted-key',
+  ['address'],
```

Comment:
A comment that didn't seem to make it into my review. This part is tightly coupled with the shape of the shape of the redux store. Perhaps the whitelisted keys should be defined next to that interface and imported here?